src/Entity/Activity/Activity.php

```diff
@@ -473,6 +473,26 @@ public function getImage(): ?EmbeddedFile
         return $this->image;
     }
 
+    /**
+     * Get imageUpdatedAt.
+     *
+     * @return \DateTime
+     */
+    public function getImageUpdatedAt(): ?\DateTime
+    {
+        return $this->end;
```

Comment:
This seems wrong?

Reply:
correct, should be `this->ImageUpdatedAt`